revolv/payments/signals.py

```diff
@@ -0,0 +1,146 @@
+from django.db.models import Sum, signals
+from django.dispatch import receiver
+from revolv.base.models import RevolvUserProfile
+from revolv.payments.models import (AdminReinvestment, AdminRepayment, Payment,
+                                    PaymentType, Repayment)
+from revolv.payments.utils import (NotEnoughFundingException,
+                                   ProjectNotCompleteException)
+
+
+@receiver(signals.pre_init, sender=AdminRepayment)
+def pre_init_admin_repayment(**kwargs):
+    """
+    Make sure that related project is indeed complete, else throw a
+    ProjectNotCompleteException and to disallow instantiation of an invalid
+    AdminRepayment.
+    """
+    init_kwargs = kwargs.get('kwargs')
```

Comment:
It's not immediately obvious to me why not having a `kwargs` kwarg would mean that there's not enough funding, could we have a comment explaining that or something?